sdk_container/.repo/manifests/mantle-container

```diff
@@ -1 +1 @@
-ghcr.io/flatcar/mantle:git-d638477431717aae4d87909d5ee5e39f009a803f
+ghcr.io/flatcar/mantle:git-9eef5e9f00c307469b1cd7e6ad276075fc8b6695
```

Comment:
This commit can be dropped

Reply:
If I remove it then some tests will fail in my branch :) I'll rebase to main before merging anyway so this will be gone before we merge.